core/src/main/scala/org/apache/spark/api/java/JavaSparkContext.scala

```diff
@@ -670,6 +670,19 @@ class JavaSparkContext(val sc: SparkContext)
   }
 
   /**
+   * Add a file to be downloaded with this Spark job on every node.
+   * The `path` passed can be either a local file, a file in HDFS (or other Hadoop-supported
+   * filesystems), or an HTTP, HTTPS or FTP URI.  To access the file in Spark jobs,
+   * use `SparkFiles.get(fileName)` to find its download location.
+   *
+   * A directory can be given if the recursive option is set to true. Currently directories are only
+   * supported for Hadoop-supported filesystems.
+   */
+  def addFile(path: String, recursive: Boolean): Unit = {
```

Comment:
Oh, I see. I found ```_jsc.sc()``` and ```_jsc``` are mix-used in ```context.py```. I will do some clean up and unify them in a follow up work. Thanks for your comments.